libs/account-lib/src/types/dtos/revokeDelegation.request.dto.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable max-classes-per-file */
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { HexString } from '@polkadot/util/types';
+import { TransactionType } from '../enums';
+import { ISubmittableResult, Signer, SignerPayloadJSON, SignerPayloadRaw, SignerResult } from '@polkadot/types/types';
+
+class RevokeDelegationPayloadDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  providerId: string;
+}
+
+export class RevokeDelegationPayloadResponseDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  accountId: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
```

Comment:
Consider `@IsNumberString()`

Reply:
(normally don't need validators on a response type, but since the request extends this class, it's good to have)

Reply:
I'll fix all these, I would suggest not spending too much on these.